apps/additional_apps.sh

```diff
@@ -2,10 +2,10 @@
 
 # T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
 
-# shellcheck disable=2034,2059
+# shellcheck disable=2015,2034,2059
 true
 # shellcheck source=lib.sh
-. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+[ -f /var/scripts/main/lib.sh ] && source /var/scripts/main/lib.sh || . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
```

Comment:
@enoch85, if you don't like that I had to disable SC2015 in order to fix travis, instead of writing it like this: 
```
[ -f /var/scripts/main/lib.sh ] && source /var/scripts/main/lib.sh || . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
```
We could also write it like this:
```
[ -f /var/scripts/main/lib.sh ] && source /var/scripts/main/lib.sh
[ ! -f /var/scripts/main/lib.sh ] . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
``
Which solution do you like more? (We could remoce the disabling of SC2015 afterwards again)

Reply:
What about

```
if [ -f /var/scripts/main/lib.sh ]
then
    source /var/scripts/main/lib.sh
else
    source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
fi
```

Never tested with `source` for curl, but `.` means the same...

Reply:
hm, I think it is a bit too long to be placed on the beginning of every file; I prefer a one-liner.
What about:
```
if [ -f /var/scripts/main/lib.sh ]; then source /var/scripts/main/lib.sh; else source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh); fi
```

btw: `source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)` seems to work!

Reply:
Please make the decision, then I change it accordingly.